src/main/commandServer/httpCommandServer.ts

```diff
@@ -89,7 +116,7 @@ export class HttpCommandServer {
 
         return;
       }
-      await command.call(this, request, response);
+      await command.call(this, request, response, { interactive: user === 'interactive' });
```

Comment:
Would be clearer if the variable name is `userType` or even `currentUserType` and not `user` -- I initially assumed you misspelled `interactive-user` as `interactive`.